aws/assets/ssl.conf

```diff
@@ -0,0 +1,52 @@
+<IfModule mod_ssl.c>
```

Comment:
https://gist.github.com/RajithaKumara/87a69a196691083829ce4057dac47bdc#file-orangehrm-recommended-server-config-md
`X-frame-options` need to consider